index.php

```diff
@@ -0,0 +1,4 @@
+<?php
+if (in_array($needle, $haystack)) {
```

Comment:
> Specify 3rd parameter explicitly when calling in_array to avoid unexpected comparison results.
>
> [link](http://localhost:3000/gh/114079365/pull_requests/6)

test